### PR TITLE
build(cargo): update swc_core, dependencies to resolve circular deps

### PR DESCRIPTION
**Pull request by kwonoj**
Fixes WEB-155

This is the final piece in turbopack resolves circular deps with indexmap. This can go next.js with swc_core bump.

Prerequisites:

- https://github.com/swc-project/plugins/pull/114
- https://github.com/wooorm/mdxjs-rs/pull/7

### CODE REVIEW

**Comment by kdy1** [comment]
@kwonoj Seems like both are merged

**Comment by kwonoj** [comment]
Yup, will wrap up this soon.

File: crates/turbopack-core/Cargo.toml [hunk 1]
```diff
@@ -10,27 +10,27 @@ autobenches = false
...
-serde = { version = "1.0.136", features = ["rc"] }
-serde_json = { version = "1.0.85", features = ["preserve_order"] }
-serde_regex = "1.1.0"
-sourcemap = "6.0.2"
-swc_core = { workspace = true, features = ["ecma_preset_env", "common"] }
-tokio = "1.21.2"
-turbo-tasks = { path = "../turbo-tasks" }
-turbo-tasks-env = { path = "../turbo-tasks-env" }
-turbo-tasks-fs = { path = "../turbo-tasks-fs" }
+anyhow           = "1.0.47"
```
Comment (wbinnssmith) [inline crates/turbopack-core/Cargo.toml:13]: reformat?

Comment (kwonoj) [reply crates/turbopack-core/Cargo.toml:13]: huh I don't know why did this happen 🤔

Comment (kwonoj) [reply crates/turbopack-core/Cargo.toml]: Updated!

**Review by ForsakenHarmony — APPROVED** [review]

**Review by wbinnssmith — APPROVED** [review]

**Pull request merged**